### PR TITLE
Release notes SCXA 9 March 2020

### PR DESCRIPTION
**Pull request by pcm32**
Release notes SCXA 9 March 2020

Numbers to be updates

@alfonsomunozpomer could you guys add any relevant software feature on this PR's branch? Thanks

To update list of experiments:
```
curl 'https://wwwdev.ebi.ac.uk/gxa/sc/json/experiments' | jq -r '.experiments | .[] | select(.lastUpdate | strptime("%d-%m-%Y") | mktime > 1577750400) | [.experimentAccession, .experimentDescription] | @csv' | awk -v FS="," '{ printf "- [%s](https://www.ebi.ac.uk/gxa/sc/experiments/%s)\n", $2, $1}' | sed s/\"//g
```

### CODE REVIEW

**Comment by anjaf** [comment]
Thanks @pcm32 
As the curator on duty today, here are my comments:
- It shows " 2,139,182 cells" on wwwdev right now, this doesn't match with the numbers here.
- I would add the Latin species name "marmoset (Callithrix jacchus)"
- The first two experiments have the same title. Why is the 10x/dropseq missing at the end?
- Shall we label the 8 HCA datasets, e.g. (HCA) after the title?  I don't like the idea much myself but it's just that we say we have them but it will be hard to actually find which ones are the HCA experiments. I assume the HCA landing page isn't updated yet.
- Then is the question, how do we count HCA datasets? We now say 11 up from 3 (I guess that is based on the EHCA/HCAD accessions) but the old HCA landing page already has 7 studies. Because these 7 include other GEO and MTAB experiments that have been associated with the HCA.

**Comment by pcm32** [comment]
Thanks for looking at this @anjaf !

> It shows " 2,139,182 cells" on wwwdev right now, this doesn't match with the numbers here.

This is because the web application needs to re-start to pick up the numbers from the `cell-stats.json`. The number you see are taken from that file. You can see these numbers as well in the fallback instances, that were restarted yesterday.

> The first two experiments have the same title. Why is the 10x/dropseq missing at the end?

I don't know, this is retrieved automatically from the web application, which in turn retrieve it I understand it from the IDF files. So maybe this is something that curators need to check on the IDF files. But let me know if it looks different there.

> Shall we label the 8 HCA datasets, e.g. (HCA) after the title? I don't like the idea much myself but it's just that we say we have them but it will be hard to actually find which ones are the HCA experiments. I assume the HCA landing page isn't updated yet.

I'm on my way to removing any HCA explicit mention, as I was told by Irene yesterday that we should not announce HCA datasets until after they are released.

> Then is the question, how do we count HCA datasets? We now say 11 up from 3 (I guess that is based on the EHCA/HCAD accessions) but the old HCA landing page already has 7 studies. Because these 7 include other GEO and MTAB experiments that have been associated with the HCA.

For the same reason I guess that this won't be a problem right now, but it would be good to sort it out for the announcement at the end of the month. Currently I was counting everything with HCA in the accession.

**Review by anjaf — APPROVED** [review]

File: HowToGetNumbersAndSections.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ Top 15 species by studies count overall in atlas (including single cell expressi
...
 "
 
-all_species=`curl -s https://wwwdev.ebi.ac.uk/gxa/json/experiments | jq '.aaData | .[].species' | sort -u`
+all_species=`curl -s https://wwwdev.ebi.ac.uk/gxa/json/experiments | jq '.experiments | .[].species' | sort -u`
 
 for species in $all_species; do
-	 differential_studies=`curl -s https://wwwdev.ebi.ac.uk/gxa/json/experiments | jq '.aaData | map(select(.species | contains('"$species"'))) | map(select(.experimentType | test("(MICROARRAY)|(DIFFERENTIAL)"; "i"))) | length'`
-	 baseline_studies=`curl -s https://wwwdev.ebi.ac.uk/gxa/json/experiments | jq '.aaData | map(select(.species | contains('"$species"'))) | map(select(.experimentType | test("(BASELINE)"; "i"))) | length'`
-	 singlecell_studies=`curl -s https://wwwdev.ebi.ac.uk/gxa/sc/json/experiments | jq '.aaData | map(select(.species | contains('"$species"'))) | map(select(.experimentType | test("(BASELINE)"; "i"))) | length'`
+	 differential_studies=`curl -s https://wwwdev.ebi.ac.uk/gxa/json/experiments | jq '.experiments | map(select(.species | contains('"$species"'))) | map(select(.experimentType | test("(MICROARRAY)|(DIFFERENTIAL)"; "i"))) | length'`
```
Comment (suhaibMo) [inline HowToGetNumbersAndSections.md]: @pcm32 `.experimentType` -> `.rawExperimentType` in the updated response. This field needs changes where ever used.

Comment (pcm32) [reply HowToGetNumbersAndSections.md]: so the last time I did it just now it works with .experimentType (changed from .aaData - but only did it for SC, are you referring only to bulk Suhaib?). @alfonsomunozpomer could you comment on what is the correct usage now? .experimentType or .rawExperimentType? I don't see this as crucial for this release notes now though, but of course we want to be using the correct way going forward.

Comment (pcm32) [reply HowToGetNumbersAndSections.md]: or is it .rawExperimentType for bulk and .experimentType for SC? and why would this be?

Comment (suhaibMo) [reply HowToGetNumbersAndSections.md]: For both bulk and single cell.
- bulk
e.g    `"rawExperimentType": "RNASEQ_MRNA_BASELINE",`
- single cell
 e.g `"rawExperimentType": "SINGLE_CELL_RNASEQ_MRNA_BASELINE",`

Comment (pcm32) [reply HowToGetNumbersAndSections.md]: ok, I underdstand now, will do, thanks @suhaibMo

Comment (pcm32) [reply HowToGetNumbersAndSections.md]: changed now, let me know if it is good to merge.

**Pull request merged**